build/toolchain/host/BUILD.gn

```diff
@@ -32,3 +32,13 @@ gcc_toolchain("${host_os}_${host_cpu}_clang") {
     is_clang = true
   }
 }
+
+gcc_toolchain("fake_${host_cpu}_gcc") {
+  toolchain_args = {
+    current_os = host_os
+    current_os = host_os
+    current_cpu = host_cpu
+    is_clang = true
```

Comment:
fake gcc is clang? Why is this important  and should it not let it be default (which is likely false)?

Reply:
Please fix.

Reply:
done.

Reply:
It still says gcc but is_clang=true

Reply:
Nevermind I was confused by the GitHub UI.